src/tests/integration/connection/connectionV2Accept.spec.ts

```diff
@@ -0,0 +1,244 @@
+    .getPrivate()
+    .toString('hex');
+
+  if (sourcePrivateKey.length !== 64) {
+    sourcePrivateKey =
+      '22f8f182a11f90e1a64d693c7cbf44ca017c50b6efc51445ea2f319210875c09';
+  }
+
+  // Generate random username
+  let username = `User${Math.random()
+    .toString(36)
+    .substring(7)}`;
+
+  let sourceUserWalletId: string;
+  let targetUserId: string;
+  let pSdk: PillarSdk;
+
+  const responseData = {
+    result: 'success',
+    message: 'Connection invitation accepted',
+  };
+
+  const errInvalidWalletId = {
+    message: 'Could not find a Wallet ID to search by.',
+  };
+
+  const errInternal = {
+    message: 'Internal Server Error',
+  };
+
+  beforeAll(async () => {
+    pSdk = new PillarSdk({});
+    pSdk.configuration.setUsername('username');
+
+    if (env === 'test') {
+      const mockApi = nock('https://localhost:8080');
+      mockApi
+        .post('/register/keys')
+        .reply(200, {
+          expiresAt: '2015-03-21T05:41:32Z',
+          nonce: 'AxCDF23232',
+        })
+        .post('/register/auth')
+        .reply(200, {
+          authorizationCode: 'Authorization code',
+          expiresAt: '2011-06-14T04:12:36Z',
+        })
+        .post('/register/access')
+        .reply(200, {
+          accessToken: 'accessToken',
+          refreshToken: 'refreshToken',
+          walletId: 'targetWalletId',
+          userId: 'targetUserId',
+        })
+        .post('/register/keys')
+        .reply(200, {
+          expiresAt: '2015-03-21T05:41:32Z',
+          nonce: 'AxCDF23232',
+        })
+        .post('/register/auth')
+        .reply(200, {
+          authorizationCode: 'Authorization code',
+          expiresAt: '2011-06-14T04:12:36Z',
+        })
+        .post('/register/access')
+        .reply(200, {
+          accessToken: 'accessToken',
+          refreshToken: 'refreshToken',
+          walletId: 'sourceWalletId',
+          userId: 'sourceUserId',
+        })
+        .post('/connection/v2/accept')
+        .reply(200, responseData)
+        .post('/connection/v2/accept')
+        .reply(400, errInvalidWalletId)
+        .post('/connection/v2/accept')
+        .reply(500, errInternal)
+        .post('/connection/v2/accept');
+    }
+
+    let walletRegister = {
+      privateKey: targetPrivateKey,
+      fcmToken: '987qwe1',
+      username,
+    };
+
+    let response = await pSdk.wallet.registerAuthServer(walletRegister);
+    targetUserId = response.data.userId;
+
+    username = `User${Math.random()
+      .toString(36)
+      .substring(7)}`;
+
+    walletRegister = {
+      privateKey: sourcePrivateKey,
+      fcmToken: '987qwe2',
+      username,
+    };
+
+    response = await pSdk.wallet.registerAuthServer(walletRegister);
+    sourceUserWalletId = response.data.walletId;
+  });
+
+  afterAll(() => {
+    jest.restoreAllMocks();
+    if (env === 'test') {
+      nock.cleanAll();
+    }
+  });
+
+  it('expects to return a success message and status 200', async () => {
```

Comment:
i think we should execute this tests only if env === 'test'. Because if we execute against local platform-core, the response is going to be an error